server/mongodb/actions/meeting.ts

```diff
@@ -0,0 +1,27 @@
+import { connectToDB } from "../index";
+import MeetingSchema from "../Meeting";
+import { ObjectId } from "mongodb";
+
+export async function addMeeting(meeting: Document) {
+  await connectToDB();
+
+  return MeetingSchema.create(meeting);
+}
+
+export async function getMeetings() {
+  await connectToDB();
+
+  return MeetingSchema.find().sort({ startDateTime: -1 });
+}
+
+export async function getMeetingById(id: ObjectId) {
+  await connectToDB();
+
+  return MeetingSchema.findOne({ _id: id });
```

Comment:
Replace this with `findById(id)`. If the parameter passed to `findOne` is null, then it returns a random document.

Reply:
Should I make this change for applications.ts too since I did the same thing there?